dev/bots/prepare_package.dart

```diff
@@ -54,7 +54,9 @@ class PreparePackageException implements Exception {
 
 enum Branch {
   beta,
-  stable;
+  stable,
+  master,
+  main;
```

Comment:
Do we need both master and main?

Reply:
Yes for now, dart-internal is using main but the development workflows are using master.